### ISSUE TITLE
Wrong migration history order

### ISSUE THREAD

**Issue by evgen-d** [yiisoft/yii2#12980]
console config:
```php
'controllerMap' => [
    'migrate' => [
        'class' => 'yii\console\controllers\MigrateController',
        'migrationNamespaces' => [
            'app\migrations',
            'app\modules\mymodule\migrations',
        ],
        'migrationPath' => null,
    ],
],
```
apply some migrations:
```
app\migrations\M140506102106One
app\migrations\M160909083544Two
app\migrations\M160930135248Three
app\modules\mymodule\migrations\M161018124749Four
app\modules\mymodule\migrations\M161025123028Five
app\migrations\M161110133341Six
```
run `./yii migrate/history`
```
Showing the last 6 applied migrations:
    (2016-11-12 16:53:04) app\migrations\M161110133341Six
    (2016-11-12 16:53:04) app\modules\mymodule\migrations\M161025123028Five
    (2016-11-12 16:53:03) app\modules\mymodule\migrations\M161018124749Four
    (2016-11-12 16:52:20) app\migrations\M160930135248Three
    (2016-11-12 16:52:17) app\migrations\M160909083544Two
    (2016-11-12 16:52:16) app\migrations\M140506102106One
```
then run `./yii migrate/history 1`

### What is the expected result?
```
Showing the last 1 applied migration:
    (2016-11-12 16:53:04) app\migrations\M161110133341Six
```
### What do you get instead?
```
Showing the last 1 applied migration:
    (2016-11-12 16:53:04) app\modules\mymodule\migrations\M161025123028Five
```

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | master
| PHP version      | 7.0
| Operating system | Fedora 24

**Comment by SilverFire** [comment]
@evgen-d would you like to fix this one?

**Comment by evgen-d** [comment]
@SilverFire i think that `version` and `namespace` must be on different columns of DB table, but this can break BC

**Comment by SilverFire** [comment]
@yiisoft/core-developers

**Comment by samdark** [comment]
Seems to be related to https://github.com/yiisoft/yii2/issues/12974?

**Comment by evgen-d** [comment]
It related to, but not the result of this issue

**Comment by klimov-paul** [comment]
Resolved by commit 1157657cab3c3ef867a5bd929b893e44f5f2dbcc
